### ISSUE TITLE
Book 05: Sentence 076

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#7]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/79db62c4-4ca9-4c23-9ec8-ab116d7c7fdf)

```
# sent_id = Book_05_-_The_Omens.txt:76
# text = Ca ie no iuhquj, muchiuhquj: qujcenujca, qujcemjtoa in mjqujztetzaujtl.
# text[norm] = Za ye no iuhqui, mochiuhqui: quicenhuica, quicemihtoa in miquiztetzahuitl.
# text[orig] = Caienoiuhquj·,·muchiuh¶quj·:·qujcenujca·,·qujcemjtoa·in·mj¶qujztetzaujtl·.
# text[osp] = [...] o que los imbidiosos, y maliuolos los hechauan dentro de casa, por mal querencia: y por hazer mal a los mordores, deseando los enfermedad, o muerte, o pobreza, y desasosiego.
# references = A&D/175:4
1	Ca	za	ADV	_		8	_	_	Orig=Caienoiuhquj|Folio=10v|Paragraph=4|Line=26|Norm=Za|AmbigNorm=True|Override=True|Analysed=Yes
2	ie	ye	ADV	_	_	8	_	_	Orig=Caienoiuhquj|Folio=10v|Paragraph=4|Line=26|Norm=ye|AmbigNorm=True|Override=True
3	no	no	ADV	_		8	_	_	Orig=Caienoiuhquj|Folio=10v|Paragraph=4|Line=26|Norm=no|Gloss=también|Analysed=Yes
4	iuhquj	iuhqui	ADV	_		8	_	_	Orig=Caienoiuhquj|Folio=10v|Paragraph=4|Line=26|Norm=iuhqui|Gloss=así|Analysed=Yes
5	,	,	PUNCT	_		8	punct	_	Folio=10v|Paragraph=4|Line=26|Norm=,|Analysed=Yes
6	muchiuhquj	chihua	VERB	_	_	8	_	_	Orig=muchiuh¶·quj|Folio=10v,10v|Paragraph=4,4|Line=26,27|Norm=mochiuhqui|Ambiguous=Yes|Analysed=Yes
7	:	:	PUNCT	_		8	punct	_	Folio=10v|Paragraph=4|Line=27|Norm=:|Analysed=Yes
8	qujcenujca	cenhuica	VERB	_	Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Pres|VerbForm=Fin	0	root	_	Folio=10v|Paragraph=4|Line=27|Norm=quicenhuica|Analysed=Yes
9	,	,	PUNCT	_		10	punct	_	Folio=10v|Paragraph=4|Line=27|Norm=,|Analysed=Yes
10	qujcemjtoa	cemihtoa	VERB	_	Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Pres|VerbForm=Fin	8	_	_	Folio=10v|Paragraph=4|Line=27|Norm=quicemihtoa|Analysed=Yes
11	in	in	DET|SCONJ	_	_	12	_	_	Folio=10v|Paragraph=4|Line=27|Norm=in
12	mjqujztetzaujtl	miquiztetzahuitl	NOUN	_	State=Abs	8	_	_	Orig=mj¶·qujztetzaujtl|Folio=10v,10v|Paragraph=4,4|Line=27,28|Norm=miquiztetzahuitl|Split=miquiz·tetzahuitl|Analysed=Yes
13	.	.	PUNCT	_		8	punct	_	Folio=10v|Paragraph=4|Line=28|Norm=.|Analysed=Yes
```

**Comment by ftyers** [comment]
Probably *mochiuhqui* wants to be the `root`, but what form is it?

López Austín:
> Ya también así sucede: plenamente lleva, plenamente manifiesta augurio de muerte ella, la rana casera.

[Note tha LA has "in yehhuatl calcuiyatl" as part of this sentence, whereas in the [image](https://www.loc.gov/resource/gdcwdl.wdl_10616/?sp=22&st=image&r=0.473,0.513,0.656,0.346,0) it is part of the subsequent sentence.]

**Comment by Lguyogiro** [comment]
So I think "-qui" can form a type of resultative/stative word, like a participle. I'm reminded of "molonqui", "boiled" (from moloni to boil) in nhi.

maybe something like "Así se ha hecho"?

**Comment by Lguyogiro** [comment]
- here, for `quicenuica, quisemijtoa in miquiztetzahuitl`, I feel like the two verbs should have a conj relation. mainly because they share the obj, `miquiztetzahuitl`.

**Comment by Lguyogiro** [comment]
![imagen](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/7957321/a9f5d22a-8a4f-463c-8a0e-957f44ce42d8)

something like this?

**Comment by Lguyogiro** [comment]
Yeah I think in the case that there is a single explicit realization of a shared argument, we should add a subrelation (`parataxis:shared` or whatever we end up thinking is a better name) so that we can find those cases later. Other than that, parataxis makes sense here as well.

**Comment by ftyers** [comment]
How about:
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/33bf46dd-bde7-4ec4-b4ae-bea7f97c68cc)

What's the motivation behind having *no* hang off *iuhqui* ?

**Comment by Lguyogiro** [comment]
LGTM

> What's the motivation behind having no hang off iuhqui ?

I think just bias due to the joining of those two elements in e.g. nhi. the analysis above is good.

**Comment by ftyers** [comment]
Fixed in 2194905
